geometry_msgs/PoseArray[] pose_array
